docs/DEVELOPMENT.md

```diff
@@ -2,12 +2,12 @@
 
 ## Getting started
 
-1. [Ramp up on kubernetes and CRDs](https://github.com/kubesphere/s2ioperator/blob/s2i-dev-guide/docs/DEVELOPMENT.md#ramp-up-on-kubernetes-and-crds)
-2. [Checkout your fork](https://github.com/kubesphere/s2ioperator/blob/s2i-dev-guide/docs/DEVELOPMENT.md#checkout-your-fork)
-3. [Set up your develop environment](https://github.com/kubesphere/s2ioperator/blob/s2i-dev-guide/docs/DEVELOPMENT.md#set-up-local-develop-environment)
-4. [Iterating](https://github.com/kubesphere/s2ioperator/blob/s2i-dev-guide/docs/DEVELOPMENT.md#iterating)
-5. [Test](https://github.com/kubesphere/s2ioperator/blob/s2i-dev-guide/docs/DEVELOPMENT.md#test)
-6. [Install CRD](https://github.com/kubesphere/s2ioperator/blob/s2i-dev-guide/docs/DEVELOPMENT.md#install-s2i-crd)
+1. [Ramp up on kubernetes and CRDs](https://github.com/kubesphere/s2ioperator/blob/master/docs/DEVELOPMENT.md#ramp-up-on-kubernetes-and-crds)
```

Comment:
Prefer use relative paths

like `[Ramp up on kubernetes and CRDs] (DEVELOPMENT.md#ramp-up-on-kubernetes-and-crds)`